pkg/scope/health/health.go

```diff
@@ -0,0 +1,148 @@
+/*
+Copyright 2020 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package health
```

Comment:
This package also should be part of https://github.com/crossplane/addon-oam-kubernetes-local/pull/48, right?

Reply:
I am following the convention for traits and workloads here. There is a discussion to refactor all of these outside this package into a new one.

Reply:
That is different from the "core type" discussion. The existing trait/workload implementations are in  crossplane/addon-oam-kubernetes-local. The containerized/manualScalar code in this repo are just utility code.

Reply:
Didn't we have a recent discussion to merge it all into a single repo?

Reply:
it's a different issue, we won't solve this until we solve the "remote" repo issue